content/_data/changelogs/lts.yml

```diff
@@ -5257,6 +5257,55 @@
     message: |-
       Winstone 5.16: Update Jetty from 9.4.38.v20210224 to 9.4.39.v20210325 for bug fixes and enhancements.
 
+- version: "2.277.4"
+  date: 2021-05-05
+  changes:
+  - type: major bug
```

Comment:
not sure if I would class this as major but 🤷

Reply:
I agree

```suggestion
  - type: bug
```